### PR TITLE
executor: support Chunk for DeallocateExec

### PR DESCRIPTION
**Pull request by zz-jason**
to #5261

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@tiancaiamao PTAL

File: executor/prepared.go [hunk 1]
```diff
@@ -238,23 +238,22 @@ type DeallocateExec struct {
...
-		return nil, errors.Trace(plan.ErrStmtNotFound)
+		return errors.Trace(plan.ErrStmtNotFound)
 	}
 	delete(vars.PreparedStmtNameToID, e.Name)
 	delete(vars.PreparedStmts, id)
-	return nil, nil
-}
-
-// Close implements Executor Close interface.
-func (e *DeallocateExec) Close() error {
```
[the remark this answers is not in the document]
Comment (coocood) [reply executor/prepared.go]: It is implemented by baseExecutor.

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**